app/containers/SendTips/CurrencyField.jsx

```diff
@@ -124,19 +126,19 @@ const CurrencyField = ({
           <Label>
             {translationMessages[locale]?.[messages.sendFromAccount.id]}
           </Label>
-          {!isPeer && (
-            <B onClick={selectAccount} type="button">
-              {(isScatterWalletSelected || isKeycatWalletSelected) && (
+          {!isPeer &&
+            !(withScatter && isScatterWalletSelected) &&
```

Comment:
enums would definitely help simplify this logic. This comparison would be as simple as ` currentWallet == selectedWallet`